web/src/components/tables/UsersTable.tsx

```diff
@@ -92,6 +92,13 @@ export const UsersTable = () => {
         emptyText: "No users",
         apiPaginator: paginator,
         linkTo: true,
+        rowProps: (row) => ({
+          "_hover": { bg: "gray.50" },
+          "cursor": "pointer",
+          // in our test cases we use reason for the unique key
```

Comment:
comment doesn't match
Do we actually need all the row props like _hover ect to be specified? I though it would have been applied already 
```suggestion
          // in our test cases we use reason for the unique key
```

Reply:
In my local branch the hover and pointer are already defaults